### PR TITLE
[Nataliya Mykytyuk] Feature flags

### PR DESCRIPTION
**Pull request by dnistrova**

### CODE REVIEW

**Review by dverbovyi — CHANGES_REQUESTED** [review]
@dnistrova good stuff.

* Please add unit-test for KillswitchService
* This feature requires documentation. Please create doc describing how to use create/update and use the killswitch in the app

File: src/app/core/services/killswitch.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  it('should return promise', () => {
+    const killswitchService: KillswitchService = TestBed.get(KillswitchService);
+    const subject = killswitchService.loadConfig();
+    const isPromise = typeof subject.then === 'function';
+    expect(isPromise).toBe(true);
+  });
+
+  it('should return corect value of object', () => {
```
Comment (dverbovyi) [inline src/app/core/services/killswitch.service.spec.ts:25]: your test doesn't check anything

use the next
```
killswitchService.killswitches = {'wishListEnabled': true};
expect(killswitchService.getKillswitch('wishListEnabled')).toBe(true);
```

**Review by dverbovyi — CHANGES_REQUESTED** [review]
fix comment

update README

File: src/app/core/services/killswitch.service.spec.ts (hunk 1 above)
Comment (dnistrova) [reply src/app/core/services/killswitch.service.spec.ts:25]: variable killswitches is private

**Review by dverbovyi — APPROVED** [review]

**Comment by dverbovyi** [comment]
@dnistrova ready to merge once the conflicts have been solved

File: src/app/home-page/components/home-page/home-page.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<app-slideshow></app-slideshow>
+<!-- <app-slideshow></app-slideshow>
```
Comment (dverbovyi) [inline src/app/home-page/components/home-page/home-page.component.html]: what's that?

**Review by dverbovyi — CHANGES_REQUESTED** [review]
@dnistrova looks like merges went not well

File: src/app/home-page/components/home-page/home-page.component.html (hunk 2 above)
Comment (dnistrova) [reply src/app/home-page/components/home-page/home-page.component.html]: fixed

**Review by dverbovyi — APPROVED** [review]

**Pull request merged**